### ISSUE TITLE
How to use and What is the difference with GUI?

### ISSUE THREAD

**Issue by arlovibre** [sinedie/SRTranslator#66]
Hello. I normally use SRTranslatorGUI, but I am having some problems with it. It is not working smoothly. What is the difference between yours and GUI? When something is not working with GUI, does this mean there is something wrong with DeepL or Firefox that affects your version as well? And as I am a non-technical user, I could not manage to get your version to work. I have anaconda3 on my pc, how do I use your SRTranslator? Sorry for asking a lot, I am just trying to learn. Thank you for your great work.

**Comment by sinedie** [comment]
Hey, the GUI is a wrapper around this library -I didn't make it so don't know details about its implementation-.

Errors on traslation could be cause of proxies, firefox, deepl, changes on the deepl page or this package, etc. GUI can have an outdated version of this package and is most probably the issue. Try to open an issue in @KryptoST package to fix it.

The easy way to use this package is from the command line (windows CMD). Just need to copy this (and change the path to the file and the source and target language)

`python -m srtranslator ./filepath/to/srt -i en -o es`

Check [here](https://github.com/sinedie/SRTranslator/blob/master/docs/deepl_scrap.md#supported-languages) for the supported languages and docs.

**Comment by arlovibre** [comment]
> Hey, the GUI is a wrapper around this library -I didn't make it so don't know details about its implementation-.
> 
> Errors on traslation could be cause of proxies, firefox, deepl, changes on the deepl page or this package, etc. GUI can have an outdated version of this package and is most probably the issue. Try to open an issue in @KryptoST package to fix it.
> 
> The easy way to use this package is from the command line (windows CMD). Just need to copy this (and change the path to the file and the source and target language)
> 
> `python -m srtranslator ./filepath/to/srt -i en -o es`
> 
> Check [here](https://github.com/sinedie/SRTranslator/blob/master/docs/deepl_scrap.md#supported-languages) for the supported languages and docs.

Thank you so much for quick and detailed reply, and your patience. Best wishes to you.

**Comment by arlovibre** [comment]
> Try to reinstall it, I made a mistake. It is fixed by now on version 0.3.1

I did what you had told me, but now I am getting: OSError: [Errno 22] Invalid Argument

or when I try to write the line like this: python -m srtranslator C:\Users\...\Masaüstü\ID1099373.srt/to/srt -i en -o tr
instead of like this: python -m srtranslator ./C:\Users\...\Masaüstü\ID1099373.srt/to/srt -i en -o tr
I am getting this error: FileNotFoundError: [Errno 2] No such file or directory. 

What am I doing wrong exactly?

**Comment by arlovibre** [comment]
> Try to reinstall it, I made a mistake. It is fixed by now on version 0.3.1

I did what you had told me, but now I am getting: OSError: [Errno 22] Invalid Argument

or when I try to write the line like this: python -m srtranslator C:\Users...\Masaüstü\lll.srt/to/srt -i en -o tr

instead of like this: python -m srtranslator ./C:\Users...\Masaüstü\lll.srt/to/srt -i en -o tr

I am getting this error: FileNotFoundError: [Errno 2] No such file or directory.

What am I doing wrong exactly?

**Comment by sinedie** [comment]
dont mix \ with /

**Comment by arlovibre** [comment]
I am getting this error now.
![image](https://github.com/sinedie/SRTranslator/assets/141356815/2ec65037-4c56-4884-aa73-28e2ed77b02a)

**Comment by sinedie** [comment]
sure that file/folder exists? seems like it doesnt

**Comment by arlovibre** [comment]
> sure that file/folder exists? seems like it doesnt

I am sure it exists. I tried putting it directly into C folder, but the same error again...

**Comment by sinedie** [comment]
did you put the extension in the filepath?

**Comment by arlovibre** [comment]
> did you put the extension in the filepath?

Could you please explain what this means? 
This is what I am writing: 
python -m srtranslator C:/Users/mutta/Masaüstü/lll.srt/to/srt -i en -o tr

**Comment by sinedie** [comment]
srt is a file or a folder?

**Comment by arlovibre** [comment]
> srt is a file or a folder?

It is a file. Like this:
![image](https://github.com/sinedie/SRTranslator/assets/141356815/d035ecc4-fdad-43d0-970a-78e6e4ffcd9d)
I am using Aegisub to edit it. That's why it's symbol is an eye.

**Comment by arlovibre** [comment]
Here is the file type:
![image](https://github.com/sinedie/SRTranslator/assets/141356815/c86b29e1-1055-438a-9d05-9ba29ea2aca3)

**Comment by sinedie** [comment]
`python -m srtranslator C:/Users/mutta/Masaüstü/lll.srt/to/srt.srt -i en -o tr`

need the `.srt` extension

**Comment by arlovibre** [comment]
I tried this, did not work again. The same error. 😐

**Comment by sinedie** [comment]
Can't help you more beyond that. The issue is not this package, the system won't find that file -thats a python error, not srtranslator error-.

The path must be wrong or maybe the special characters in Masaüstü break it.

**Comment by arlovibre** [comment]
Hello, it's me again. I tried writing Desktop instead of Masaüstü, and I tried all of this with another pc as well, and it did not work again. Both pcs have anaconda3, and we both try to do this from there. Could this be an anaconda issue, if so what else I can use? I've read that phyton sometimes does not see system files or something like this, and tried the fixes they recommended, but it did not work again.
